src/software/test_util/test_util.h

```diff
@@ -135,6 +136,32 @@ namespace Test
         static ::testing::AssertionResult equalWithinTolerance(
             const Circle &c1, const Circle &c2, double tolerance = METERS_PER_MILLIMETER);
 
+        /**
+         * Checks if two Angles are within tolerance of each other
+         *
+         * @param a1, a2 Angles to compare
+         * @param tolerance tolerance to check equality with
+         *
+         * @return AssertionSuccess if the two Angles are within tolerance of each other
+         */
+        static ::testing::AssertionResult equalWithinTolerance(const Angle &a1,
+                                                               const Angle &a2,
```

Comment:
Worth moving these functions and tests to an equal_within_tolerance file (still within test_util though)

Reply:
out of scope for this PR? (easy to do just might as well try keep things organized here since this one's already kinda big). But yea I agree.

Reply:
made #1425